.travis.yml

```diff
@@ -1,37 +1,34 @@
 language: python
 
-sudo: required
-dist: trusty
+cache: pip
 
-python:
-  - "2.6"
-  - "2.7"
-  - "3.4"
+dist: trusty
 
-#matrix:
-#  allow_failures:
-#    - python: "3.4"
+matrix:
+  include:
+    - python: "2.6"
+      env: TOXENV=py26-sqlite
+    - python: "2.7"
+      env: TOXENV=py27-sqlite
+    - python: "3.4"
+      env: TOXENV=py34-sqlite
+    - python: "3.5"
+      env: TOXENV=py35-sqlite
 
 addons:
   apt:
     packages:
-      - libgeos-c1
+      - libgeos-dev
+      - libpq-dev
       - libxml2-dev
-
-before_script:
-  - pycsw-admin.py -c get_sysprof
+      - libxslt1-dev
+      - libz-dev
 
 install:
-  - pip install -r requirements.txt
-  - pip install -r requirements-dev.txt
-  - pip install -r requirements-standalone.txt
-  - python setup.py -q install
+  - pip install tox
```

Comment:
That is not possible. The way in which `tox` works is that it is responsible for creating separate virtualenvs for each python version (and other combinations of dependencies, like one virtualenv for python3.4 with sqlite, another virtualenv for python3.4 with postgresql, etc). As such, the tox process is the one which is going to be calling pip to install the dependencies inside each virtualenv after it has been created. This happens when we run `tox` on the command-line. This means that tox must be installed system-wide and *a priori* of installing pycsw rquirements.

In short, its tox's job to install the requirements, so tox has to be already installed and there is no benefit in including it in `requirements.txt`.